source/delegate/contracts/Delegate.sol

```diff
@@ -366,13 +366,14 @@ contract Delegate is IDelegate, Ownable {
     uint256 priceCoef,
     uint256 priceExp
   ) internal {
-    rules[senderToken][signerToken] = Types.Rule({
+    rules[senderToken][signerToken] = Rule({
       maxSenderAmount: maxSenderAmount,
       priceCoef: priceCoef,
       priceExp: priceExp
     });
 
     emit SetRule(
+      owner(),
```

Comment:
is `owner` or `tradeWallet` more likely to be the thing people want to know?

Reply:
good question, my thought was ruleOwner allows for a user / staker to track their delegate whereas tradeWallet is more about fund and could be queries from getting the ruleOwner and querying a delegateFactory.